### ISSUE TITLE
Count the number of words and duplicates in a text by using map

### ISSUE THREAD

**Issue by JunJul** [sobhanbera/algorithms#181]
#### This issue is about [Algo/DS Name/Question Name](link to resource for the Algo/DS/Question)

-   [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [ ] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [ ] I will send my PR only after I'm assigned to this issue by any maintainer

-   [ ] Assign this issue to me (I want to work on this)

-   [ ] I ensure that I am not already assigned to 2 or more issues

**Comment by JunJul** [comment]
Please add a Hacktoberfest label for me @jhudsonsg

**Comment by JunJul** [comment]
We can use two maps to count the word. One map is used to count the total number of words. Another map is used to count the duplicates.The functions of the map are O(1) so they are quick.

**Comment by sobhanbera** [comment]
This change belongs to the other section of this project.

**Comment by vanshikaa-agarwal** [comment]
I want to work on this

**Comment by ShrutikaK2911** [comment]
I want to work on this

**Comment by sobhanbera** [comment]
Hi There!

You guys can create a new issue. As this issue is currently assigned to @JunJul.

**Comment by JunJul** [comment]
I already finished the issue. You can close it.@sobhanbera
